### DISCUSSION TITLE
Is there a way to close the datepicker overlay when the date is chosen?

### DISCUSSION THREAD

**Discussion by davestewart** [unovue/reka-ui#1480] [Q&A] [answered]
The system datepicker calendar closes when a date is chosen.

Radix's does not.

How do we achieve this?

Right now I am adding a ref and "fake" clicking it:

```ts
// ref to hidden close component
const close = ref<DatePickerClose>(null)

// later
close.value.$el.click()
```

**Comment by zernonia** [comment, marked as answer]
Responded in https://github.com/unovue/radix-vue/discussions/1479#discussioncomment-11486991 😁

**Reply by adlermore** [reply]
👍 NICEEE
